doc/03_reference/exception_interrupts.rst

```diff
@@ -4,17 +4,18 @@ Exceptions and Interrupts
 =========================
 
 Ibex implements trap handling for interrupts and exceptions according to the RISC-V Privileged Specification, version 1.11.
+(Is it fully compatible with `RISC-V Privileged Specification, version 1.12 <https://drive.google.com/file/d/1EMip5dZlnypTk7pt4WWUKmtjUKTOkBqh/view>`?).
```

Comment:
There is always some confusion about version of RISC-V specs, but I do not believe this is correct.  According to the [CV32E20 Specification](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/CVE20/CV32E20_Specification_v0_1.pdf) (not to be confused with this User Manual), the target privileged specification for the CV32E20 is `The RISC-V Instruction Set Manual, Volume II: Privileged Architecture, Document Version 20211203`, which I believe is v.1.13.

Reply:
I would gladly put a reference to the correct instruction set manual if you point me to the correct version

Reply:
The spec for the above version of the priv spec has been added to the [RISC-V Technical Specifications Archive](https://wiki.riscv.org/display/HOME/RISC-V+Technical+Specifications+Archive) ([link](https://drive.google.com/file/d/1EMip5dZlnypTk7pt4WWUKmtjUKTOkBqh/view)).

Reply:
I see, the CV32E20 points to The ``RISC-V Instruction Set Manual, Volume II: Privileged Architecture, Document Version 20211203``, but I couldn't find somewhere a phrase in the doc that said that the version of the ISA manual was v1.xy .
For now I will put the reference to the link you provided

Reply:
updated